### ISSUE TITLE
Error 1020 when trying to share my webcam

### ISSUE THREAD

**Issue by Steffen75** [bigbluebutton/bigbluebutton#15720]
**Describe the bug**
When trying to share the webcam using a current Firefox on macOS I'm getting a 1020 connection error lately with my own server. It has worked before. Also sharing the webcam using https://test.bigbluebutton.org/ from this browser works fine.

Audio works fine on my server though.

The current versions are:

```
bbb-conf --version

BigBlueButton Server 2.5.5 (3091)

ii  bbb-apps-akka                         2.5-31                            all          BigBlueButton Apps (Akka)
ii  bbb-config                            1:2.5-55                          amd64        BigBlueButton configuration utilities
ii  bbb-etherpad                          1:2.5-11                          amd64        The EtherPad Lite components for BigBlueButton
ii  bbb-freeswitch-core                   2:2.5-11                          amd64        BigBlueButton build of FreeSWITCH
ii  bbb-freeswitch-sounds                 1:2.5-8                           amd64        FreeSWITCH Sounds
ii  bbb-fsesl-akka                        2.5-25                            all          BigBlueButton FS-ESL (Akka)
ii  bbb-html5                             1:2.5-2846                        amd64        The HTML5 components for BigBlueButton
ii  bbb-learning-dashboard                1:2.5-21                          amd64        BigBlueButton bbb-learning-dashboard
ii  bbb-libreoffice-docker                1:2.5-6                           amd64        BigBlueButton setup for LibreOffice running in docker
ii  bbb-mkclean                           1:2.5-5                           amd64        Clean and optimize Matroska and WebM files
ii  bbb-pads                              1:2.5-13                          amd64        BigBlueButton Pads
ii  bbb-playback                          1:2.5-9                           amd64        BigBlueButton playback
ii  bbb-playback-presentation             1:2.5-11                          amd64        BigBluebutton playback of presentation
ii  bbb-record-core                       1:2.5-27                          amd64        BigBlueButton record and playback
ii  bbb-web                               1:2.5-39                          amd64        BigBlueButton API
ii  bbb-webrtc-sfu                        1:2.5-26                          amd64        BigBlueButton WebRTC SFU
ii  bigbluebutton                         1:2.5-2                           amd64        Open source web conferencing platform (bbb)
ii  libopusenc0                           0.2.1-1bbb1                       amd64        High-level API for encoding Ogg Opus audio streams
ii  yq                                    3.4.1bbb1                         amd64        lightweight and portable command-line YAML processor
```

And when you check the status:
```
bbb-conf --status

nginx —————————————————► [✔ - active]
freeswitch ————————————► [✔ - active]
redis-server ——————————► [✔ - active]
bbb-apps-akka —————————► [✔ - active]
bbb-fsesl-akka ————————► [✔ - active]
mongod ————————————————► [✔ - active]
bbb-html5 —————————————► [✔ - active]
bbb-webrtc-sfu ————————► [✔ - active]
kurento-media-server ——► [✔ - active]
bbb-html5-backend@1 ———► [✔ - active]
bbb-html5-backend@2 ———► [✔ - active]
bbb-html5-frontend@1 ——► [✔ - active]
bbb-html5-frontend@2 ——► [✔ - active]
etherpad ——————————————► [✔ - active]
bbb-web ———————————————► [✔ - active]
bbb-pads ——————————————► [✔ - active]
bbb-rap-caption-inbox —► [✔ - active]
bbb-rap-resque-worker —► [✔ - active]
bbb-rap-starter ———————► [✔ - active]
```

Also a bbb-conf --check reports no errors.

But when I restart the server I can see the following

```
bbb-conf --restart
Restarting BigBlueButton 2.5.5 ...
Stopping BigBlueButton

Applying updates in /etc/bigbluebutton/bbb-conf/apply-config.sh: 
  - Enable Firewall and opening 22/tcp, 80/tcp, 443/tcp and 16384:32768/udp
Skipping adding existing rule
Skipping adding existing rule (v6)
Skipping adding existing rule
Skipping adding existing rule (v6)
Skipping adding existing rule
Skipping adding existing rule (v6)
Firewall is active and enabled on system startup

Starting BigBlueButton
Too few arguments.

# Warning:  could not be started. Please, check BBB-LTI or BBB-Demo.
#     Run the command:
#       sudo journalctl -u 
#     To better understand the ERROR


# Potential problems described below
...........
```
What could be the "Too few arguments"?

**Comment by prlanzarin** [comment]
Prefer using the mailing lists for debugging setup problems. Re-open if this turns out to be an actual issue (which seems unlikely since it works on test*).

**Comment by Steffen75** [comment]
The point of making a bug report out of this was "it has worked before". And a self destructing application like BBB in this case pretty much feels like a bug to me. This was very likely not a setup problem.

- The original installation of BBB was done using 2.5.2 or 2.5.3 and everything was working fine (Can't remember which version it was back then.)
- After the installation the server only gets updated using apt and an occasional reboot when the Ubuntu tells me to do so
- Updates up to including 2.5.4 have worked fine
- After the update to 2.5.5 webcam use was broken with a 1020 connection error

Interestingly before doing a clean install I tried a last update via apt and BBB was updated to 2.5.6 . And after this the line "Too few arguements" was gone. But webcams still wouldn't work.

Did a clean installation with 2.5.6 on a fresh Ubuntu with my usual set of parameters and BBB works again.
